component/appcat_sla_reporter.jsonnet

```diff
@@ -6,11 +6,13 @@ local kube = import 'lib/kube.libjsonnet';
 local inv = kap.inventory();
 // The hiera parameters for the component
 local params = inv.parameters.appcat;
-local sla_reporter_params = params.slos.sla_reporter;
+local slos_params = params.slos;
+local sla_reporter_params = slos_params.sla_reporter;
+local mimir_endpoint = 'http://' + sla_reporter_params.slo_mimir_svc + '.' + sla_reporter_params.slo_mimir_namespace + '.svc.cluster.local:8080/prometheus';
```

Comment:
`slo_mimir_svc` is currently the full URL in the defaults.yaml file on line 115

Reply:
ups, will have to fix that, also will have to update the code for SLA job